AdyenActions/Components/Base/ActionComponent.swift

```diff
@@ -19,7 +19,7 @@ public protocol ActionComponent: Component {
 /// :nodoc:
 public protocol ParameterlessInitializable {
```

Comment:
its not `Parameterless` any more :-D, right?

Reply:
how about `APIContextInitializable`?

Reply:
yeah, sounds good to me.